### PR TITLE
Introduce poetry and github actions

### PR DESCRIPTION
**Pull request by hirosassa**
## What is this PR?
 
I added followings:

- `poetry` for package management
- `github actions` for CI (travis is closing soon)
- add badges on README

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[tox]
+envlist = py37
```
Comment (Hi-king) [inline tox.ini:2]: [ASK] In pyproject.toml `python = "^3.6"` is declared. Maybe you can test this with py36?

Comment (hirosassa) [reply tox.ini:2]: Sorry, this is my mistake (this is for test on python 3.6 but failed).
Fied!

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+documentation = ""
+
+[tool.poetry-dynamic-versioning]
+enable = true
+style = "pep440"
+pattern = "^(?P<base>\\d+\\.\\d+\\.\\d+)"
+
+[tool.poetry.dependencies]
+python = "^3.7"
+luigi = "*"
```
Comment (vaaaaanquish) [inline pyproject.toml:19]: Maybe you don't need to write it, it depends on gokart.

**Comment by vaaaaanquish** [comment]
🆒

**Review by vaaaaanquish — APPROVED** [review]

File: pyproject.toml (hunk 2 above)
Comment (hirosassa) [reply pyproject.toml]: You're right! fixed!

File: pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+[tool.poetry.dev-dependencies]
+tox = "*"
+moto = "*"
+testfixtures = "*"
+coverage = "*"
+
+[build-system]
+requires = ["poetry"]
+build-backend = "poetry.masonry.api"
```
Comment (Hi-king) [inline pyproject.toml:38]: @hirosassa [ASK] Now we don't need setup.py, right?

**Review by Hi-king — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 3 above)
Comment (hirosassa) [reply pyproject.toml:38]: Thanks, Removed!

**Review by Hi-king — APPROVED** [review]
@hirosassa LGTM!

**Comment by Hi-king** [comment]
https://github.com/m3dev/redshells/runs/2026959704
works fine!

**Pull request merged**